clang/include/clang/Driver/Options.td

```diff
@@ -5152,7 +5152,8 @@ def : Flag<["-"], "nocudalib">, Alias<nogpulib>;
 def gpulibc : Flag<["-"], "gpulibc">, Visibility<[ClangOption, CC1Option]>,
   HelpText<"Link the LLVM C Library for GPUs">;
 def nogpulibc : Flag<["-"], "nogpulibc">, Visibility<[ClangOption, CC1Option]>;
-def nodefaultlibs : Flag<["-"], "nodefaultlibs">;
```

Comment:
I think these options are primarily for ELF where some may apply to Mach-O.

These options likely don't make sense for `dxc.exe` from Windows.

Reply:
Ok, I'll remove them. It is easy enough to add them later on. I just don't want to tack on too many options if it does not make any sense and/or serves no purpose.